### PR TITLE
clarify text, add links to new SAML docs

### PR DESCRIPTION
**Pull request by yvonnewilson**
Added links to a couple other new SAML doc pages into this page, so it can serve as one-stop shopping for SAML info.

### CODE REVIEW

File: docs/saml-configuration.md [hunk 1]
```diff
@@ -49,18 +49,32 @@ Auth0 can be configured as a Service Provider to any other SAML-compliant Identi
 
 Configuring Auth0 to serve as a SAML Identity Provider is done in a couple different places, depending on the type of application.
 
-For some 3rd party applications that support SAML, the Auth0 side of the configuration is done using the "Third Party Apps" link in the dashboard, and selecting the specific application.  Instructions specific to each application are provided.
+For some Third Party applications that support SAML, the Auth0 side of the configuration is done using the "Third Party Apps" link in the dashboard, clicking on the "NEW THIRD PARTY APP" button, and selecting the specific application.  Instructions specific to the chosen application are provided.
```
Comment (dschenkelman) [inline docs/saml-configuration.md]: why caps for third party?

Comment (dschenkelman) [inline docs/saml-configuration.md]: I assume because the page is "Third Party Apps"

File: docs/saml-configuration.md [hunk 2]
```diff
@@ -49,18 +49,32 @@ Auth0 can be configured as a Service Provider to any other SAML-compliant Identi
...
 
-For any application not listed on the "Third Party Apps" page, the Auth0 side of the configuration is done using the "Apps/APIs" link and then clicking on the "Addons" tab.
+* [Third Party SAML Web Apps](@@env.BASE_URL@@/saml2webapp-tutorial)
+
+
+For any application not listed on the "Third Party Apps" page, (click NEW THIRD PARTY APP button to see the list) the Auth0 side of the configuration can be done using the "Apps/APIs" link and then clicking on the "Addons" tab for the particular application, and clicking on the toggle in the "SAML2 WEB APP" box.
```
Comment (dschenkelman) [inline docs/saml-configuration.md]: remove "button"

File: docs/saml-configuration.md (hunk 1 above)
Comment (yvonnewilson) [inline docs/saml-configuration.md]: Yes, I tried to make the text in the docs match what is on the screen.

File: docs/saml-configuration.md (hunk 2 above)
Comment (yvonnewilson) [inline docs/saml-configuration.md]: ok, thanks, will do.

**Pull request merged**